dev_guide/builds.adoc

```diff
@@ -519,9 +531,42 @@ For example, defining a custom HTTP proxy to be used during build:
 ----
 ====
 
+[[source-code]]
+
+== Git Repository Source Options
```

Comment:
the diff is already pretty unreadable, i resorted to viewing the current state and trusting you hadn't accidentally deleted/modified something unrelated.